webviews/src/views/SceneGraphInspectorView/NodeDetailPage.svelte

```diff
@@ -17,8 +17,24 @@
         let nodeTree = inspectNodeNodeTree;
         if (inspectNodeNodeTree) {
             while (nodeTree) {
-                if (nodeTree.id) {
-                    absoluteKeyPathParts.unshift(nodeTree.id);
+                if (nodeTree.subtype === 'RowListItem') {
```

Comment:
I don't fully understand why we're modifying the keypath structure here. Shouldn't we keep the full keypath as RTA provides?